packages/fxa-shared/db/config.ts

```diff
@@ -402,8 +402,9 @@ export function makeRedisConfig() {
         doc: `Password for connecting to Redis`,
       },
       prefix: {
-        default: 'recovery-phone:',
-        env: 'RECOVERY_PHONE_REDIS_KEY_PREFIX',
```

Comment:
@vpomerleau I think this was the footgun that was giving you the weird look up results in redis when searching for keys. i.e. You had to change `sms-attempt:*` to `*:sms-attempt:*` to get things working locally. I think for all intents and purposes we don't want to rely on this prefix and just have a fixed one that is defined by the recovery phone library itself. See how the exported variable RECOVERY_PHONE_REDIS_PREFIX is now used to keep things consistent throughout our code.

Reply:
Much cleaner throughout!